SDL_opengl.h dosen't have the nessecary headers. GL/gl.h has the parts
of SDL_opengl.h needed for SDL_opengl_glext.h, then include
SDL_opengl_glext.h which has the *f functions.

--- src/client/renderer/r_gl.h  2020-10-08 07:50:29.818581990 +1100
+++ src/client/renderer/r_gl.h  2020-10-08 07:52:16.805006133 +1100
@@ -29,7 +29,8 @@
 #include "../../ports/android/r_gles_android.h"
 #include <stdlib.h>
 #else
-#include <SDL_opengl.h>
+#include <GL/gl.h>
+#include <SDL_opengl_glext.h>
 #endif

 inline void R_DrawArrays (GLint first, GLsizei count) {

